prov/efa/src/rxr/rxr_init.c

```diff
@@ -429,6 +429,28 @@ int rxr_get_lower_rdm_info(uint32_t version, const char *node,
 	return ret;
 }
 
+static void rxr_shm_space_check(void)
+{
+	struct statvfs stat;
+	char *shm_fs = "/dev/shm";
```

Comment:
char shm_fs[]?

Reply:
Just different way of initialization. Updated though.